core/trino-main/src/main/java/io/trino/operator/SimplePagesHashStrategy.java

```diff
@@ -279,11 +280,12 @@ public boolean isPositionNull(int blockIndex, int blockPosition)
     public int compareSortChannelPositions(int leftBlockIndex, int leftBlockPosition, int rightBlockIndex, int rightBlockPosition)
     {
         int channel = getSortChannel();
+        checkState(comparisonOperators.get(channel).isPresent(), "type is not orderable");
```

Comment:
remove in favor of `orElseThrow` below -- this lets you check things once

Reply:
I see. Thanks.